waku/waku_sync/protocol.nim

```diff
@@ -54,25 +54,32 @@ proc serverReconciliation(self: WakuSync, message: seq[byte]): Result[seq[byte],
   ok(payload)
 
 proc clientReconciliation(
-  self: WakuSync, message: seq[byte],
+  self: WakuSync, negentropy: pointer, message: seq[byte],
```

Comment:
Don't we already have negentropy in `WakuSync` ?

Reply:
We cannot keep using same negentropy instance each time sync has to be initated. There is a preventive check in the library to avoid that. Which lead me to think that a new instance has to be created on client side everytime we want to initiate a sync. This also takes care of scenarios such as 

If we want to do parallel sync with 2 different nodes, having a single instance of negentropy on client side would not work whereas having a single one on server side should work i guess(something i want to validate as well as part of tests).

Reply:
Yes, per our 1:1 discussion we do have to create new negentropy instance for each new sync "session".